packages/clarity-js/src/interaction/encode.ts

```diff
@@ -28,26 +26,37 @@ export default async function(type: Event): Promise<void> {
         case Event.TouchMove:
         case Event.TouchCancel:
             for (let i = 0; i < pointer.state.length; i++) {
-                if (task.shouldYield(timer)) { await task.suspend(timer); }
                 let entry = pointer.state[i];
+                let pTarget = metadata(entry.data.target as Node);
                 tokens = [entry.time, entry.event];
-                tokens.push(observe(entry.data.target as TargetInfo));
+                tokens.push(pTarget.id);
                 tokens.push(entry.data.x);
                 tokens.push(entry.data.y);
+                if (pTarget.region) { tokens.push(pTarget.region); }
                 queue(tokens);
+                baseline.track(entry.event, entry.data.x, entry.data.y); // Track changes to baseline
             }
```

Comment:
since region is a id - how do you map to region name for clicks while being stateless?